src/overlays/actors/ovl_En_Sda/z_en_sda.c

```diff
@@ -68,7 +68,7 @@ Vec3f D_80947B10[] = {
     { -1.0f, -2.0f, -0.2f }, { 0.0f, -2.0f, -0.2f },  { 1.0f, -2.0f, -0.2f },  { 0.0f, -3.0f, -0.5f },
 };
 
-static s32 sPad[2] = { 0, 0 };
```

Comment:
Is this pad needed? I have seen this just be alignment for the texture in OOT.

Reply:
A quick check shows it is needed, but it can be `static s32 sPad = 0;` just fine

This particular data is at `0x80947D20` so it needs something so the vtx data doesn't snap to it

Reply:
All good catch, the one in Tanron can also just be a single `s32`. I'll make them singular in #1292.